src/main/java/frc/robot/RobotContainer.java

```diff
@@ -54,6 +62,23 @@ private void configureButtonBindings() {
     
   }
 
+  /**
+   * set the drive train type
+   * @param type
+   */
+  public void setDriveTrainType(int type) {
+    SmartDashboard.putNumber(Constants.DRIVE_TRAIN_TYPE, type);
+    driveTrainType_ = type == 0 ? DriveTrainType.TankDrive : DriveTrainType.ArcadeDrive;
```

Comment:
Conditional assignments should have paranthesis.
```suggestion
    driveTrainType_ = (type == 0) ? DriveTrainType.TankDrive : DriveTrainType.ArcadeDrive;
```
Wouldn't this be more clear if you used the enum.ordinal?
```suggestion
    driveTrainType_ = (type == DriveTrainType.TankDrive.ordinal()) ? DriveTrainType.TankDrive : DriveTrainType.ArcadeDrive;
```

Or more robustly:
```suggestion
    if (type == DriveTrainType.TankDrive) {
        driveTrainType_ = DriveTrainType.TankDrive;
    } else if (type == DriveTrainType.ArcadeDrive) {
        driveTrainType_ = DriveTrainType.ArcadeDrive;
    } else {
        throw new Exception("Unknown Drive Type");
    }
```